images/Dockerfile.py3

```diff
@@ -71,44 +77,20 @@ RUN export ASM_VERSION=1.4.7-asm.0 && \
     mv istio-${ASM_VERSION} /usr/local && \
     ln -sf /usr/local/istio-${ASM_VERSION}/bin/istioctl /usr/local/bin/istioctl
 
-    
-# Create go symlinks
-RUN ln -sf /usr/local/go/bin/go /usr/local/bin && \
-    ln -sf /usr/local/go/bin/gofmt /usr/local/bin && \
-    ln -sf /usr/local/go/bin/godoc /usr/local/bin
-
-RUN go get github.com/kelseyhightower/kube-rsa
-
-COPY ./images/checkout_repos.sh /usr/local/bin
-COPY ./images/checkout.sh /usr/local/bin
-COPY ./images/setup_ssh.sh /usr/local/bin
-RUN chmod a+x /usr/local/bin/checkout* /usr/local/bin/setup_ssh.sh
+COPY checkout_repos.sh /usr/local/bin/
+COPY checkout.sh /usr/local/bin/
+COPY setup_ssh.sh /usr/local/bin/
+COPY run_workflows.sh /usr/local/bin/
+COPY run_release.sh /usr/local/bin/
 
-COPY ./images/run_workflows.sh /usr/local/bin
-RUN chmod a+x /usr/local/bin/run_workflows.sh
-
-COPY ./images/run_release.sh /usr/local/bin
-RUN chmod a+x /usr/local/bin/run_release.sh
-
-# Install the hub CLI for git
-RUN cd /tmp && \
-    curl -LO  https://github.com/github/hub/releases/download/v2.11.2/hub-linux-amd64-2.11.2.tgz && \
-    tar -xvf hub-linux-amd64-2.11.2.tgz && \
-    mv hub-linux-amd64-2.11.2 /usr/local && \
-    ln -sf /usr/local/hub-linux-amd64-2.11.2/bin/hub /usr/local/bin/hub
+RUN chmod a+x /usr/local/bin/checkout* /usr/local/bin/setup_ssh.sh /usr/local/bin/run_*.sh
 
 # Install kubectl
 # We don't install via gcloud because we want 1.10 which is newer than what's in gcloud.
 RUN  curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.14.0/bin/linux/amd64/kubectl && \
     mv kubectl /usr/local/bin && \
     chmod a+x /usr/local/bin/kubectl
 
-RUN go get -u github.com/jstemmer/go-junit-report
-
-# Create a cached copy of the python test scripts so that we don't
-# need to clone the repo just to get access to them
-RUN mkdir -p /srcCache/kubeflow/testing
-COPY py /srcCache/kubeflow/testing/py
-COPY notebook_testing /srcCache/kubeflow/testing/notebook_testing
+ENV PYTHONPATH /src/kubeflow/testing/py/
```

Comment:
Do you need to set the python path here? Is run_workflows.sh depending on this? 
I don't think we want to set the python path because in some cases we will want to use 
/srcCache/kubeflow/testing
and sometimes we will want to use /src/kubeflow/testing

If run_workflows.sh needs the pythonpath it might be better to set the path there.

Reply:
@scottilee Any thoughts on how to address this?